MapboxVision/Services/VideoPlayer.swift

```diff
@@ -0,0 +1,148 @@
+//
+//  VideoPlayer.swift
+//  MapboxVision
+//
+//  Created by Alexander Pristavko on 5/17/19.
+//  Copyright © 2019 Mapbox. All rights reserved.
+//
+
+import Foundation
+import CoreMedia
+import AVFoundation
+
+protocol VideoPlayable: VideoSource {
+    var delegate: VideoPlayerDelegate? { get set }
+
+    func start()
+    func stop()
+}
+
+protocol VideoPlayerDelegate: AnyObject {
+    func playbackDidFinish()
+}
+
+private enum PlaybackConstants {
+    static let frameDuration: TimeInterval = 0.03
```

Comment:
Can we extend the value of frameDuration (probably via `1 / x` notation) to have more understanding why it equals to `0.03`?

Reply:
Ok.